code/go/0chain.net/smartcontract/dbs/event/user.go

```diff
@@ -0,0 +1,28 @@
+package event
+
+import (
+	"errors"
+
+	"0chain.net/chaincore/state"
+	"gorm.io/gorm"
+)
+
+type User struct {
+	gorm.Model
+	// Dependency: PR #759. (Default name of the forign key is `transactionID` unless struct name is changed.)
+	TransactionID string `json:"TransactionID" gorm:"transactionID"`
+
+	UserID  string        `json:"UserID" gorm:"userID"`
+	Balance state.Balance `json:"Balance" gorm:"balance"`
+	// Dependency: PR #765. (Change int64 to Nounce type.)
+	Nonce int64 `json:"Nonce" gorm:"nonce"`
```

Comment:
Not very much fluent with gorm but dont we need special annotations on foreign & primary key?